Uchu.World/Systems/Behaviors/EndBehavior.cs

```diff
@@ -12,8 +12,12 @@ public class EndBehavior : BehaviorBase
         
         public override async Task BuildAsync()
         {
-            StartAction = await GetBehavior("action");
+            StartAction = await GetBehavior("start_action");
             UseTarget = await GetParameter<int>("use_target");
         }
+        public override void ExecuteStart(BehaviorExecutionParameters parameters)
+        {
+            ((StartBehavior)StartAction).End.Invoke();
```

Comment:
Minor formatting thing: `(StartBehavior) StartAction` instead of `(StartBehavior)StartAction` (space between casting class and variable).